### ISSUE TITLE
Check counts for switzerland

### ISSUE THREAD

**Issue by sebhoerl** [matsim-eth/baseline_scenario#18]
* [ ] Run scenario synthesis with count matching location assignment
* [ ] Validate with existing count analysis tools

**Comment by ctchervenkov** [comment]
So I made a first attempt at looking at the counts for Switzerland.
street counts : https://repos2.ivt.ethz.ch/svn/ivt_studies/trunk/baseline2010/supply/base_street_counts/streetCounts_Hourly.csv
network : https://repos2.ivt.ethz.ch/svn/ivt_studies/trunk/baseline2010/supply/base_multimodal_supply/CH1903Plus_HAFAS2015/mmNetwork.xml.gz
events : 20.events.xml from a 10% Swiss scenario we were using with Joe for externalities

Then using the analysis tools in the baseline scenario repo, I compared the hourly counts.
Multiplying the simulated counts by a factor 10, this is what the comparison looks like:

![image](https://user-images.githubusercontent.com/12128010/43972947-5151fe38-9cd6-11e8-835e-7a744a5973b6.png)

In other words, NOT GOOD!
Next question is how do we fix it.

**Comment by sebhoerl** [comment]
But if it looks like this then it probably means that we distribute the vehicles to much over the network. So actually **increasing** the flow capacity might make sense, or adjusting the speeds.

**Comment by sebhoerl** [comment]
And would be interesting to see this separate for the counts of Kanton / City of Zurich / ASTRA Hgihways.

**Comment by joemolloy** [comment]
well it looks like the magnitude is way off. what about as a start comparing the total counts from the reference and scenario*factor?

**Comment by ctchervenkov** [comment]
Well, again from the results of the analysis tools that computes the hourly reference and simulation counts, I have a total of 74,067 simulated counts versus 13,121,268 reference counts, which is a factor of 177 (so with the factor=10 then that gives 17.7).
Something indeed looks fishy...

**Comment by sebhoerl** [comment]
AGAIN: We have neither cross-border traffic nor freight traffic and we lose some of the agents during the scenario synthesis (e.g. those which do not start and end a plan at "home"). So there is no reason why we should have a factor of 10.

Just find the scaling factor that minimizes some distance, e.g. L2 norm over the difference in the count stations. What matters more is the relation between the count stations, that's why I usually displayed them ordered by reference count. This will show you if "large" count stations also produce "large" counts in the simulation and the same for "small" ones.

**Comment by sebhoerl** [comment]
Have you done the checks yet whether the mapping of the count stations to the links is correct yet?
